test/DotNetty.Codecs.Tests/Base64Test.cs

```diff
@@ -0,0 +1,163 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace DotNetty.Codecs.Tests
+{
+    using System;
+    using System.Security.Cryptography.X509Certificates;
+    using System.Text;
+    using DotNetty.Buffers;
+    using DotNetty.Codecs.Base64;
+    using Xunit;
+
+    public class Base64Test
+    {
+        [Fact]
+        public void TestNotAddNewLineWhenEndOnLimit()
+        {
+            var src = Unpooled.CopiedBuffer(Encoding.ASCII.GetBytes("____abcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyzabcde"));
+            var expectedEncoded = Unpooled.CopiedBuffer(Encoding.ASCII.GetBytes("YWJjZGVmZ2hpamtsbW5vcHFyc3R1dnd4eXphYmNkZWZnaGlqa2xtbm9wcXJzdHV2d3h5emFiY2Rl"));
+            var buff = src.Slice(3, src.ReadableBytes - 3);
+            buff.ReadByte();
+            TestEncode(buff, expectedEncoded);
+        }
+
+        [Fact]
+        public void TestSimpleDecode()
```

Comment:
english is not my mother language, so it is a little difficult for me to understand what's you are talking about, especially the long text....